integration-tests/src/main/java/io/quarkus/it/amazon/lambda/LambdaResource.java

```diff
@@ -0,0 +1,87 @@
+package io.quarkus.it.amazon.lambda;
+
+import static io.quarkus.it.amazon.lambda.LambdaUtils.LAMBDA;
+import static io.quarkus.it.amazon.lambda.LambdaUtils.getPayloadBuilder;
+import static jakarta.ws.rs.core.MediaType.TEXT_PLAIN;
+
+import java.util.concurrent.CompletionStage;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+
+import org.jboss.logging.Logger;
+
+import software.amazon.awssdk.services.lambda.LambdaAsyncClient;
+import software.amazon.awssdk.services.lambda.LambdaClient;
+import software.amazon.awssdk.services.lambda.model.FunctionConfiguration;
+import software.amazon.awssdk.services.lambda.model.InvokeResponse;
+
+/*
+ * Created by triphon 21.11.23 г.
+ */
+@Path("/lambda")
+public class LambdaResource {
+    private static final Logger LOG = Logger.getLogger(LambdaResource.class);
+    public static final String HELLO_WORLD = "Hello World!";
+    public static final String OK = "OK";
+    public static final String ERROR = "ERROR";
+
+    @Inject
+    LambdaClient lambdaClient;
+
+    @Inject
+    LambdaAsyncClient lambdaAsyncClient;
+
+    @Inject
+    LambdaUtils lambdaUtils;
+
+    FunctionConfiguration functionInfo;
+
+    @PostConstruct
+    public void init() {
+        LOG.info("LambdaResource init");
+        functionInfo = lambdaUtils.findDevLambda();
+    }
+
+    @GET
+    @Path("blocking")
+    @Produces(TEXT_PLAIN)
+    public String testBlocking() {
+        LOG.info("Testing Blocking Lambda client with lambda: " + LAMBDA);
+
+        try {
+            final InvokeResponse response = lambdaUtils.invokeLambda(functionInfo);
```

Comment:
Simplify the test, we just need to assert the client can actually talk to the aws control plane and not testing the lambda itself. I feel like the test will take too much times.

Reply:
Addressed